history-of-the-quakers/original/05-book-1-chapter-2.adoc

```diff
@@ -225,7 +231,7 @@ and liberty being granted for any to speak, after some time of conference and re
 a woman present put this Question, "`What birth was that which the Apostle speaks of,
 A being born again of incorruptible seed, by the word of God,
 which liveth and abidith forever?`"^
-footnote:[1 Peter 1: 23.]
+footnote:[1 Peter 1:23.]
 The priest replied, I suffer not a woman to speak in the church,
 which drew from George Fox an inquiry what he understood the Church to be, for reply,
 the priest returning the question George expressed Explains his
```

Comment:
Github is not letting me leave a comment on it, but let's add a "book-title" ref on line 243 - to Fox's journal again.

Also on line 541 - let's do the same for "history of the Puritans" -- and maybe capitalize the "H" while you're at it.

Also, line 633 - another ref to Fox's journal